### PR TITLE
OMIC-related env variable adjustments

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: notm/dev/main.tf [hunk 1]
```diff
@@ -130,9 +129,9 @@ resource "azurerm_app_service" "app" {
     NODE_ENV                       = "production"
     API_ENDPOINT_URL               = "https://noms-api-dev.dsd.io/elite2api/"
     KEYWORKER_API_URL              = "https://keyworker-api-dev.hmpps.dsd.io/"
+    OMIC_UI_URL                    = "https://omic-dev.hmpps.dsd.io/"
     USE_API_GATEWAY_AUTH           = "yes"
     NOMS_TOKEN                     = "${data.external.vault.result.noms_token}"
-    NOMS_PRIVATE_KEY               = "${data.external.vault.result.noms_private_key}"
```
Comment (Mjwillis) [inline notm/dev/main.tf]: Need to remove the Vault reference too

Comment (steverendell) [reply notm/dev/main.tf]: ok. And no i havnt got terraform yet - I will need to actually install it all then !

Comment (Mjwillis) [reply notm/dev/main.tf]: @steverendell I can run it for you to validate if it helps....

**Comment by glenjamin** [comment]
This is going to conflict with the changes Simon's making in https://github.com/ministryofjustice/digital-studio-infra/pull/157 unfortunately.

Probably best to wait until we land that

**Comment by Mjwillis** [comment]
@glenjamin and @simoncross747 These changes are all applies to the dev/stage env but not pre-prod or prod

**Comment by glenjamin** [comment]
@steverendell @Mjwillis Could you have another look into what comfig is needed here?

We've re-arranged the repo quite a bit so might be worth starting afresh

Sorry!

**Comment by simoncross747** [comment]
The changes in the conflicted files listed above are in master now but need checking.

**Comment by steverendell** [comment]
Ive now checked the 4 conflicted keyworker-api main.tf's which are fine, and have merged the latest master into the other 4 (NOTM) .
Re key vaults, I just get 'You are unauthorized to view these contents' !

**Pull request merged**